_common/qiskit/benchmark_runner.py

```diff
@@ -93,21 +93,20 @@
 
     args = parser.parse_args()
 
-# For Inserting the Noise model default into exec option as it is function call 
+    # For Inserting the Noise model default into exec option as it is function call 
```

Comment:
A few minor things:

- "Inserting" -> "insertng"
- "Noise model" should probably be stylized as `noise_model`
- Same as "exec option" -> `exec_option`.